src/Command.ts

```diff
@@ -70,8 +70,8 @@ export default abstract class Command {
   public action(): CommandAction {
     const cmd: Command = this;
     return function (this: CommandInstance, args: CommandArgs, cb: () => void) {
-      cmd._debug = args.options.debug || false;
-      cmd._verbose = cmd._debug || args.options.verbose || false;
+      cmd._debug = args.options.debug || process.env.OFFICE365CLI_DEBUG === '1' || false;
```

Comment:
Other base classes for commands, such as `SpoCommand` don't call `super.action()` so this modifier is not being used since other services have their own implementation of `action()`. For this to work, it would have to be implemented in other base classes as well.